selfdrive/ui/qt/offroad/settings.cc

```diff
@@ -205,6 +205,18 @@ DevicePanel::DevicePanel(SettingsWindow *parent) : ListWidget(parent) {
 
   // offroad-only buttons
 
+  auto tskBtn = new ButtonControl(tr("Toyota Security Key Manager"), tr("RUN"));
+  connect(tskBtn, &ButtonControl::clicked, [this, tskBtn]() {
+    if (ConfirmationDialog::confirm(tr("Run Toyota Security Key Manager and reboot?"), tr("RUN"), this)) {
+      tskBtn->setEnabled(false);
+      QProcess *process = new QProcess(this);
+      // This shell script launches TSK Manager on top of OP and then reboots,
+      // so there's no need to handle hiding and restoring OP application.
+      process->start("/data/openpilot/tsk/manager.sh");
+    }
+  });
+  addItem(tskBtn);
+
```

Comment:
Add a disclaimer or warning of what's happening, I think it would be appreciated before the user continues

Reply:
I was hoping `"Run Toyota Security Key Manager and reboot?"` conveys that message. Any suggestions?

Reply:
That works. We can always update if there's a better version.